testing_and_setup/compass/manage_regression_suite.py

```diff
@@ -533,11 +780,23 @@ def main ():  # {{{
         # If setting up, set up the suite
         if args.setup:
             print("\n")
-            print("Setting Up Test Cases:")
-            testcases = get_test_case_procs(suite_root)
-            setup_suite(suite_root, args.work_dir, args.model_runtime,
-                        args.config_file, args.baseline_dir, args.verbose)
-            summarize_suite(testcases)
+            testcase_data_prereq = []
+            data_dic = {}
+            testcase_data, testcase_data_prereq = get_test_case_procs(suite_root, testcase_data, testcase_data_prereq)
+            testcase_data = summarize_suite(testcase_data)
+            print("\n\nSetting Up Test Cases:")
+            if args.local_parallel:
+                data_dic["procs"] = []
+                data_dic["run_file_names"] = []
+                data_dic["prereq_commands"] = []
+                data_dic["allocation_commands"] =[]
+                local_parallel_script, local_parallel_code = local_parallel_setup_script(args, suite_root, testcase_data, testcase_data_prereq, data_dic) 
+                local_parallel_script.write(local_parallel_code)
+            else:
+                regression_script, regression_script_code = setup_suite(
+                    suite_root, args.work_dir, args.model_runtime,
+                    args.config_file, args.baseline_dir, args.verbose, args)
```

Comment:
This call seems to be missing the `data_dic` argument:
```
Traceback (most recent call last):
  File "./manage_regression_suite.py", line 839, in <module>
    main()
  File "./manage_regression_suite.py", line 796, in main
    regression_script, regression_script_code = setup_suite(
TypeError: setup_suite() missing 1 required positional argument: 'data_dic'
```

Reply:
Actually, it looks like `setup_suite()` doesn't do anything with `data_dic` so the easier solution is to remove that argument from the function and the other call to it.